aicsimageio/readers/default_reader.py

```diff
@@ -37,13 +43,21 @@ def data(self) -> np.ndarray:
     @property
     def dims(self) -> str:
         """
-        Remove n number of characters from dimension order where n is number of dimensions in image data.
+        `imageio.imread` returns the first YX plane of an image, except in the case where the image is RGB / RGBA, in
+        which case it returns the first YXC plane of an image.
 
-        That said these dimensions are probably incorrect because jpg for example is actually 'XYC' dimension order,
-        but it's our assumption to make. You can override the dims by using the property setter.
+        This dims property handles those cases by making assumptions about the dimension order based off the shape of
+        the data stored in the reader. In the case where more data than an YXC plane is returned, it uses the
+        `guess_dim_order` function. However, I can't tell when that should ever get run and is really just used as a
+        safety catch all.
         """
         if self._dims is None:
-            self._dims = self.guess_dim_order(self.data.shape)
+            if len(self.data.shape) == 2:
+                self._dims = "YX"
+            elif len(self.data.shape) == 3:
+                self._dims = "YXC"
```

Comment:
just confirming: do unit tests verify that this is YXC and not XYC?  (I thought imageio might be returning XYC)

Reply:
They do confirm that the data is reshaped and read properly. But yea, I tried loading the example files from the resources directory with base `imageio` to double check. `example.png` is returned as `(800, 537, 4)` which is `YXC` for example.